 1) Fixes bug in eldap.erl, which was masked by erlang 1:11.b.1 and
    earlier versions.

Index: eldap/eldap.erl
===================================================================
--- ejabberd/src/eldap/eldap.erl
+++ ejabberd/src/eldap/eldap.erl
@@ -713,7 +713,7 @@
 	    check_id(S#eldap.id, Msg#'LDAPMessage'.messageID),
 	    case Msg#'LDAPMessage'.protocolOp of
 		{bindResponse, Result} ->
-		    case Result#'LDAPResult'.resultCode of
+		    case Result#'BindResponse'.resultCode of
 			success -> bound;
 			Error   -> {fail_bind, Error}
 		    end
